Packs/Core/Integrations/CortexCoreIR/CortexCoreIR.py

```diff
@@ -173,6 +177,136 @@ def get_asset_details_command(client: Client, args: dict) -> CommandResults:
     )
 
 
+def core_execute_command_reformate__readable_output(script_res: list) -> str:
+    """
+    Reformat the human-readable output of the 'core_execute_command' command
+    so that each command appears as a separate row in the table.
+
+    Args:
+        script_res (list): The result from the polling command.
+
+    Returns:
+        str: Reformatted human-readable output
+    """
+    reformated_results = []
+    for response in script_res:
+        results = response.outputs.get('results')
+        for res in results:
+            # for each result, get only the data we want to present to the user
+            reformatted_result = {}
+            for key in EXECUTE_COMMAND_READABLE_OUTPUT_FIELDS:
+                reformatted_result[key] = res.get(key)
+            # the command name in the output start with _
+            reformatted_result['command'] = reformatted_result['command'].removeprefix('_')
+            reformated_results.append(reformatted_result)
```

Comment:
It would be better to check if the prefix exists before removing it, unless the removeprefix considers that case. 

```suggestion
            # Remove the underscore prefix from the command name
            command_name = reformatted_result['command']
            if command_name.startswith('_'):
                reformatted_result['command'] = command_name.removeprefix('_')
            reformated_results.append(reformatted_result)
```